lib/timeline/locales.js

```diff
@@ -29,15 +31,17 @@ exports['nl_BE'] = exports['nl'];
 require('moment/locale/de');
 exports['de'] = {
   current: 'Aktuelle',
-  time: 'Zeit'
+  time: 'Zeit',
+  deleteSelected: 'L\u00f6sche Auswahl',
```

Comment:
`Gewählten Eintrag löschen`

Reply:
Why?
This is taken from https://github.com/visjs/vis-network/blob/505ef4e9aa5c500a7597523c0eed7f0ad27d45c0/lib/network/locales.js#L23